### PR TITLE
`GitHub`: improved `ISSUE_TEMPLATE`

### PR DESCRIPTION
**Pull request by NachoSoto**
- Made the required fields a bit more obvious
- Changed some fields to checkboxes
- A reference to `.verbose` logs

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ A clear and concise description of what the bug is. The more detail you can prov
    3. StoreKit 2 (_disabled with `useStoreKit2IfEnabled(false)`_) (Y/N):
    4. OS version:
    5. Xcode version:
-   6. How widespread is the issue. Percentage of devices affected.
-2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
+   6. Simulator or device:
+   7. Environment (Sandbox / TestFight / Production):
```
Comment (tonidero) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Should we make these checkboxes?

**Review by tonidero — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (NachoSoto) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Good idea let me do that.

Comment (NachoSoto) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -19,13 +19,30 @@ A clear and concise description of what the bug is. The more detail you can prov
...
-   3. StoreKit 2 (_disabled with `useStoreKit2IfEnabled(false)`_) (Y/N):
-   4. OS version:
-   5. Xcode version:
-   6. How widespread is the issue. Percentage of devices affected.
-2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
-3. Steps to reproduce, with a description of expected vs. actual behavior
-4. **Other information** (e.g. stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc.)
+   3. StoreKit version:
+      - [x] StoreKit 1
+      - [ ] StoreKit 2 (_enabled with `usesStoreKit2IfAvailable(true)`_)
```
Comment (NachoSoto) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: This was still outdated 🙈

**Review by tonidero — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -19,13 +19,30 @@ A clear and concise description of what the bug is. The more detail you can prov
...
+   5. OS version:
```
Comment (vegaro) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: This should be 4

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -19,13 +19,30 @@ A clear and concise description of what the bug is. The more detail you can prov
...
+      - [ ] TestFight
+      - [ ] Production
+   10. How widespread is the issue. Percentage of devices affected.
+2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue. **Complete logs with `Purchases.logLevel = .verbose` will help us debug this issue.**
+
+```
+Logs here
+```
+
+4. Steps to reproduce, with a description of expected vs. actual behavior
```
Comment (vegaro) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: This is 3 (and the following are 1 less too)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (NachoSoto) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: Oops thanks!

**Pull request merged**